tests/unit/main/services/settings.service.test.ts

```
@@ -38,33 +95,126 @@ jest.mock('../../../../src/main/adapters', () => ({
 }));
 
 import SettingsService from '../../../../src/main/services/settings.service';
+import fs from 'fs-extra';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)